pkg/lib/docker/errors.go

```diff
@@ -56,10 +59,29 @@ func ErrorDockerPermissions(err error) error {
 	})
 }
 
-func ErrorImageInaccessible(image string, cause error) error {
+func ErrorImageDoesntExistLocally(image string) error {
+	return errors.WithStack(&errors.Error{
+		Kind:    ErrImageDoesntExistLocally,
+		Message: fmt.Sprintf("%s does not exist locally; download it with `docker pull %s` (if your registry is private, run `docker login` first)", image, image),
+	})
+}
+
+func ErrorImageInaccessible(image string, providerType types.ProviderType, cause error) error {
 	message := fmt.Sprintf("%s is not accessible", image)
+
 	if cause != nil {
-		message += "\n" + errors.Message(cause) // add \n because docker client errors are
+		message += "\n" + errors.Message(cause)
```

Comment:
We can bring back the comment, and also complete the comment? I assume we were going to say that add \n because docker client errors are **useful**'

Reply:
haha my guess was that it was supposed to be "verbose" (since it was referring to the "\n"), but I wasn't sure, so I deleted it :) Should I add "verbose", or something else?